src/app/faqs/faqs.component.ts

```diff
@@ -0,0 +1,150 @@
+import { FormBuilder, FormControl, Validators } from '@angular/forms';
+import { SideMenuSection } from '../scrollable-side-menu/scrollable-side-menu.component';
+
+interface FaqContent {
+  title: string;
+  content: string;
+  selector: string;
+}
+
+@Component({
+  selector: 'app-faqs',
+  templateUrl: './faqs.component.html',
+  styleUrls: ['./faqs.component.scss']
+})
+export class FAQSComponent {
+
+  // Height for the mat-expansion-panel-header.
+  expansionPanelHeaderHeight = '82px';
+
+  sideMenuSections: SideMenuSection[] = [
+    {
+      title: 'What is it?',
+      anchorSelector: '.what-is-it',
+      selected: true
+    },
+    {
+      title: 'Why is it important?',
+      anchorSelector: '.why-is-it-important',
+      selected: false
+    },
+    {
+      title: "Who is behind it?",
+      anchorSelector: '.who-is-behind-it',
+      selected: false
+    },
+    {
+      title: "Who can use it?",
+      anchorSelector: '.who-can-use-it',
+      selected: false
+    },
+    {
+      title: "How do I use it?",
+      anchorSelector: '.how-do-i-use-it',
+      selected: false
+    },
+    {
+      title: "Is my data private?",
+      anchorSelector: '.is-my-data-private',
+      selected: false
+    },
+    {
+      title: "Why does it only work on Twitter?",
+      anchorSelector: '.why-does-it-only-work-on-twitter',
+      selected: false
+    },
+    {
+      title: "What languages are covered?",
+      anchorSelector: '.what-languages-are-covered',
+      selected: false
+    },
+    {
+      title: "Can it detect harmful images and videos?",
+      anchorSelector: '.can-it-detect-harmful-images-and-videos',
+      selected: false
+    },
+    {
+      title: "Can it analyse direct messages (DMs)?",
+      anchorSelector: '.can-it-analyse-direct-messages-dms',
+      selected: false
+    },
+    {
+      title: "How do I give feedback?",
+      anchorSelector: '.how-do-i-give-feedback',
+      selected: false
+    },
+    {
+      title: "How do I learn about future opportunities?",
+      anchorSelector: '.how-do-i-learn-about-future-opportunities',
+      selected: false
+    }
+  ];
+
+  constructor(private formBuilder: FormBuilder,) {
+  }
+
+  faqContent: FaqContent[] = [
+    {
+      title: 'What is it?',
+      content: '<span>FeedShield helps you to take back control of your Twitter feed. It is a free web application designed for human rights defenders, such as journalists and activists, but open for anyone to use. It syncs with the user’s Twitter account, automatically recognising and flagging harmful comments. The resulting analysis helps users limit their exposure to abusive content, by allowing them to block, mute or save comments at scale. It also allows users to create reports to store or share with investigators or other watch dogs as needed. FeedShield further helps users connect with allies, from crisis counsellors on a dedicated helpline, to local experts or support organisations.</span>',
```

Comment:
Why have `span` in every `content`? Why not just put it in the `html` template?

Reply:
@kilemensi as explained [above](https://github.com/CodeForAfrica/harassment-manager/pull/33#discussion_r1033572372), some of the content have some links in them. To display the links, it would be easier to update the content as HTML.

Reply:
I could be wrong but pdating content as innerHTML doesn't necessarily require `<span>` @kelvinkipruto